apps/meteor/client/providers/CallProvider/CallProvider.tsx

```diff
@@ -71,6 +78,57 @@ export const CallProvider: FC = ({ children }) => {
 		setQueueAggregator(result.voipClient.getAggregator());
 	}, [result]);
 
+	const openRoom = (rid: IVoipRoom['_id']): void => {
+		roomCoordinator.openRouteLink('v', { rid });
+	};
+
+	const createRoom = useCallback(
+		async (caller: ICallerInfo): Promise<IVoipRoom['_id']> => {
+			if (!user) {
+				return '';
+			}
+			const { visitor } = await visitorEndpoint({
+				visitor: {
+					token: Random.id(),
+					phone: caller.callerId,
+					name: caller.callerName || caller.callerId,
+				},
+			});
+			const voipRoom = visitor && (await voipEndpoint({ token: visitor.token, agentId: user._id }));
```

Comment:
Is it possible that `visitor` is `null` or `undefined` here? If so it might be a problem for the following instructions

Reply:
Fixed. Also remove the duplication of the same code from  CallContextValue.